Utilities/CodeGeneration/EdFi.Ods.CodeGen.Tests/Approval Tests/GeneratedFileApprovalTests.Verify_All.ForScenario.Standard_Resources_Resources.generated.approved.cs

```diff
@@ -6007,7 +6007,7 @@ public override int GetHashCode()
         public bool? AdaptiveAssessment { get; set; }
 
         /// <summary>
-        /// The category of an assessment based on format and content. For example:          Achievement test          Advanced placement test          Alternate assessment/grade-level standards          Attitudinal test          Cognitive and perceptual skills test          ...
+        /// The category of an assessment based on format and content. For example:         Achievement test         Advanced placement test         Alternate assessment/grade-level standards         Attitudinal test         Cognitive and perceptual skills test         ...
```

Comment:
These changes appear to be just whitespace change in the XML documentation of generated classes,  but why did these change appear as part of this PR?

Reply:
@gmcelhanon the approval tests were failing because of these spaces. After formatting this correctly, the tests succeeded.